build/BuildSteps.cs

```diff
@@ -101,14 +102,15 @@ public static void DownloadTemplates()
 
         public static void BuildBundleBinariesForWindows()
         {
-            Settings.WindowsBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config));
+            Settings.WindowsBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config).GetAwaiter().GetResult());
         }
 
         public static void BuildBundleBinariesForLinux()
         {
-            Settings.LinuxBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config));
+            Settings.LinuxBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config).GetAwaiter().GetResult());

```

Comment:
Ideally, we'd change the orchestrator to support async flows. Not part of the scope of this PR, and this won't cause an issue for the tool.